docs/index.md

```diff
@@ -4,4 +4,6 @@ The `pmm_dump` client utility performs a logical backup of the performance metri
 
 PMM Dump allows you to share monitoring data, collected by your PMM server, with Percona Support team securely.
 
-
+!!! hint alert alert-success "Important"
+    Starting with PMM 2.41, the standalone client utility has been seamlessly integrated into PMM and is easily accessible from the main menu **> Help > PMM Dump**.
+    We highly recommend transitioning to the integrated version, unless you need to use specific functionalities like filtering or other features exclusive to the utility, or if you need to stay on PMM version 2.40 or older.
```

Comment:
```suggestion
    Starting with PMM 2.41, the standalone client utility has been seamlessly integrated into PMM and is easily accessible from the main menu **> Help > PMM Dump**.
```

Reply:
After that, you need to provide a link to the user manual describing how the integrated version works. It could be part of the PMM Dump user manual itself or, better, the PMM user manual. Good if such a manual will have screenshots.